about.html

```diff
@@ -3,24 +3,49 @@ <h2 id="rules">Rules <a href="#rules">¶</a></h2>
 <em>"Be excellent to each other"</em> is easier said than done, and means different things to different people.
 </p>
 <p>
-The following rules are a (non-exhaustive) list of behaviours that may lead to deletion of toots, silencing or suspension of accounts, at the descretion of the instance administrators.
+The following rules are a (non-exhaustive) list of behaviours that may lead to deletion of toots, silencing or suspension of accounts, at the descretion of the instance administrators, as described in our <a href="/terms">Terms</a>.
 </p>
 <p>
 Please report behaviour that bothers you. We will keep your report confidential.
 </p>
 <ul>
-<li>We do not tolerate harrassment, stalking, and doxxing.</li>
-<li>We do not tolerate racism, sexism, verbal abuse, discrimination against gender and sexual minorities, or the advocation of any of these.</li>
+<li>We do not tolerate discriminatory behaviour and content promoting or advocating the oppression of members of marginalised groups. These groups may be characterised by any of the following (though this list is naturally incomplete):
+  <ul>
+    <li>ethnicity</li>
+    <li>gender identity or expression</li>
+    <li>sexual identity or expression</li>
+    <li>physical characteristics or age</li>
+    <li>disability or illness</li>
+    <li>nationality, residency, citizen status</li>
+    <li>wealth or education</li>
+    <li>faith, or lack thereof</li>
+  </ul>
+</li>
+<li>We do not tolerate threatening behaviour, stalking, and doxxing.</li>
```

Comment:
would an explanation of doxxing be helpful? This seems internet-slangy to me. Perhaps a link to https://en.wikipedia.org/wiki/Doxing ? (also they write it with one `x`.)

Reply:
I tried to resist defining every term in use, to keep the document short and to the point, but a link won't hurt, thank you for your suggestion. Both spellings are in use, as noted by the page you linked, though ;)